bin/describe.ml

```diff
@@ -170,23 +170,47 @@ module Descr = struct
         ]
   end
 
+  module Root = struct
```

Comment:
Please add a comment at the beginning of this module (similar to the comments for its brother modules).